pkg/envoy/cell.go

```diff
@@ -363,13 +368,13 @@ func registerSecretSyncer(params syncerParams) error {
 
 	jobGroup := params.JobRegistry.NewGroup(
 		params.Health,
-		job.WithLogger(params.Slog),
+		job.WithLogger(params.Logger),
 		job.WithPprofLabels(pprof.Labels("cell", "envoy-secretsyncer")),
 	)
 
 	params.Lifecycle.Append(jobGroup)
 
-	secretSyncerLogger := params.Logger.WithField("controller", "secretSyncer")
+	secretSyncerLogger := params.Logger.With("controller", "secretSyncer")
```

Comment:
Could we use a logfield for this as well?

Reply:
what do you mean? "controller" is still a logfield, it's just a differently named function that creates the scoped logger that we use to pass down to the actual reconciliation logic.

Reply:
What Jarno means is that
```
secretSyncerLogger := params.Logger.With("controller", "secretSyncer")
```
needs to be 
```
secretSyncerLogger := params.Logger.With(logfields.Controller, "secretSyncer")
```

Reply:
thanks for the pointer :sweat_smile: :+1: 

of course - fixed